CMake/resolve_dependency_modules/fast_float.cmake

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+include_guard(GLOBAL)
+
+set(VELOX_FAST_FLOAT_VERSION v5.3.0)
```

Comment:
@assignUser @majetideepak @kgpai Would you help review build system changes?

@wypb We may need to update the docs as well: ./CMake/resolve_dependency_modules/README.md

Reply:
@mbasmanova done.